src/Umbraco.Web.UI.Client/src/views/dictionary/dictionary.edit.controller.js

```diff
@@ -102,11 +100,7 @@ function DictionaryEditController($scope, $routeParams, $location, dictionaryRes
                     });
         }
     }
-
-    function back() {
```

Comment:
How come we are removing the back button? I think it's useful to be able to return to the root. The problem here is that the route path is incorrect and hard coded, it should go to the ($routeParams.section)/dictionary/list